.env-api.sample

```diff
@@ -0,0 +1,6 @@
+ISSUER_SERVER_URL=localhost:3001
+ISSUER_API_UI_SERVER_PORT=3002
+ISSUER_API_UI_AUTH_USER=user
+ISSUER_API_UI_AUTH_PASSWORD=password
+ISSUER_API_UI_ISSUER_NAME=my issuer
+ISSUER_API_UI_ISSUER_DID=did:polygonid:polygon:mumbai:2qE9HutYM8te92FxjxA5BNXU8NCV3Ki9hRmP8ttAHY
```

Comment:
Missing `ISSUER_API_UI_ISSUER_LOGO=`

Reply:
The `ISSUER_API_UI_ISSUER_DID` is your own local one! WE need to remove this as a placeholder, e.g. `<did:polygonid:polygon:mumbai:XXXX>`